packages/core-components/src/components/DependencyGraph/DependencyGraph.tsx

```diff
@@ -31,6 +31,25 @@ import { DependencyGraphTypes as Types } from './types';
 import { Node } from './Node';
 import { Edge, GraphEdge } from './Edge';
 import { ARROW_MARKER_ID } from './constants';
+import IconButton from '@material-ui/core/IconButton';
+import FullscreenIcon from '@material-ui/icons/Fullscreen';
+import FullscreenExitIcon from '@material-ui/icons/FullscreenExit';
+import { FullScreen, useFullScreenHandle } from 'react-full-screen';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import { useTranslationRef } from '@backstage/core-plugin-api/alpha';
+import { coreComponentsTranslationRef } from '../../translation.ts';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    overflow: 'hidden',
```

Comment:
Hmmmm. Now I'm wondering what effects this could have for the experience when it's NOT in fullscreen mode since it's being added unconditionally. No ill effects at all, especially on small screens?

Reply:
Didn't notice anything special, someone could take it for a spin too :)